service/src/main/java/uk/gov/hmcts/reform/fpl/validation/interfaces/time/HasSetTime.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.validation.interfaces.time;
+
+import uk.gov.hmcts.reform.fpl.validation.validators.time.HasSetTimeValidator;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = HasSetTimeValidator.class)
+public @interface HasSetTime {
```

Comment:
As mentioned somewhere else it doesn't reflect what the validations would be doing. Maybe we could find better name.